src/pipx/venv.py

```diff
@@ -157,20 +180,31 @@ def main_package_name(self) -> str:
         else:
             return self.pipx_metadata.main_package.package
 
+    def default_create_venv_cmd(self, override_shared: bool = False) -> List[str]:
+        cmd = [self.python, "-m", "venv"]
+        if not override_shared:
+            cmd.append("--without-pip")
+        return cmd
+
     def create_venv(self, venv_args: List[str], pip_args: List[str], override_shared: bool = False) -> None:
         """
         override_shared -- Override installing shared libraries to the pipx shared directory (default False)
         """
         logger.info("Creating virtual environment")
-        with animate("creating virtual environment", self.do_animation):
-            cmd = [self.python, "-m", "venv"]
-            if not override_shared:
-                cmd.append("--without-pip")
+        with animate(f"creating virtual environment using {self.backend}", self.do_animation):
+            if self.backend == "venv":
+                cmd = self.default_create_venv_cmd(override_shared=override_shared)
```

Comment:
If this doesn't look too bulky, we can rationalise the function away; it's used only once.

```suggestion
                cmd = [self.python, "-m", "venv"]
                if not override_shared:
                    cmd.append("--without-pip")
```